### ISSUE TITLE
Unknown behaviour read > max read length

### ISSUE THREAD

**Issue by t-neumann** [t-neumann/slamdunk#10]
Currently only tcperreadpos crashes if a read is found > than a manually set max read length.
Add some checks if such behaviour is observed and throw warnings or even abort.

**Comment by AbeerMM** [comment]
Just wanted to add. I tried running count without giving the max read length, but it doesn't generate any files or errors. I had to set it manually. It seems the estimate function is not working with me.

**Comment by t-neumann** [comment]
Yeah currently the estimation is not properly working when you feed in trimmed reads so we recommend to set it manually.
